recipe/meta.yaml

```diff
@@ -25,7 +25,7 @@ build:
 
 requirements:
   build:
-    - boost 1.66.0  # only need to get conda-forge's libkml instead of defaults
+    - boost 1.66.0  # only need to get conda-forge's libkml instead of defaults
```

Comment:
That blank was interpreted as a unicode thing and broke Python 2.7 build on Windows :unamused: